landoapi/hgexportbuilder.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Module for constructing Mercurial patches in 'hg export' format.
+"""
+
+HG_EXPORT_PATCH_TEMPLATE = """# HG changeset patch
+# User {author}
+# Date {patchdate}
```

Comment:
I tested with with a patch and it works great. You're right in that we will not need the human formatted date! 

Patch that I tried via `hg import`:

```
# HG changeset patch
# User Israel Madueme <imadueme@mozilla.com>
# Date 1498600861 14400
Bug 123: Add some comments to client.py

diff --git a/client.py b/client.py
--- a/client.py
+++ b/client.py
@@ -1,15 +1,16 @@
 #!/usr/bin/python
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.

 # This is a comment to test diffs
-# This is a very unsafe thing to put in a diff
+# This is a very unsafe thing to put in a diff
+# This is stupid and should be removed
 LIBFFI_DIRS = (('js/ctypes/libffi', 'libffi'),)
 HG_EXCLUSIONS = ['.hg', '.hgignore', '.hgtags']

 CVSROOT_LIBFFI = ':pserver:anoncvs@sources.redhat.com:/cvs/libffi'

 import os
 import sys
 import datetime
```